models/cm_compartmentcurrents.cpp

```diff
@@ -50,18 +52,25 @@ nest::Na::Na( const DictionaryDatum& channel_params )
   {
     e_Na_ = getValue< double >( channel_params, "e_Na" );
   }
-  if ( channel_params->known( "V_init" ) )
+
+  // set the initialization of the channel
+  double v_init = -75.;
```

Comment:
I have an idea how to avoid this.

Reply:
fixed with commit [ca5580f](https://github.com/nest/nest-simulator/pull/2773/commits/ca5580f5f1bd7fc5ca0c8ab4c460c7f4c36d366a)